modules/java_api/src/test/java/org/intel/openvino/README.md

```diff
@@ -0,0 +1,42 @@
+## Build
+
+1. Download test data:
+```bash
+git clone https://github.com/openvinotoolkit/testdata
+```
+
+2. Create an folder `native` with native libraries in `java_api` folder. Then run `gradle` to build a package and run tests:
```

Comment:
That's done by Gradle automatically (when setupvars.sh is called).

Reply:
I think we just forgot to change these instructions